bears/uploadTool.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+from coalib.collecting.Importers import iimport_objects
+from string import Template
+
+
+def touch(file_name):
+    """
+    Creates an empty file.
+
+    :param file_name: Name of the file.
+    """
+    if not os.path.exists(file_name):
+        open(file_name, 'w').close()
```

Comment:
the behaviour of `touch` for existing files with content is that it remains, so if you want a real `touch` you should use `a` instead of `w` :)